PowerShell_Scripts/FolderSynchronisation_edited+func_2ed.ps1

```diff
@@ -0,0 +1,27 @@
+﻿Clear-Host
```

Comment:
The scripts seem to be named a bit differently from ones that already exist in the repository, probably to keep both versions there. This is not really needed thanks to git, which allows us to see all the changes made to a file - and to retrieve a file in a particular state at any moment. 😎 On the other hand, renaming files is often undesirable as it breaks git's versioning abilities - because from its perspective these are two separate files.

To keep the change history tied to the files, consider renaming them again to match the ones that already exist. 🙂

Reply:
Yeah, it's just a bit easier for me to keep the files with different names on my computer... It's more like a habit, I'll work on it 🙂